packages/vitest/src/node/cli/cli-api.ts

```diff
@@ -282,6 +331,53 @@ export function formatCollectedAsString(files: File[]) {
   }).flat()
 }
 
+export function parseFilter(filter: string): Filter {
```

Comment:
It doesn't seem like these two functions belong here because they are not used for the CLI specifically. Maybe create a separate file for them?

Reply:
What do you think of `vitest/src/node/filter.ts`?